### PR TITLE
Docs 11323 update mule default version selection pa

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

**Review by sbehrends — DISMISSED** [review]
Thanks Priya for reviewing this. 
LGTM 🚀

File: modules/ROOT/pages/proxy-deploy-raml-oas-proxy.adoc [hunk 1]
```diff
@@ -12,10 +12,12 @@ NOTE: OpenAPI Specification (OAS) 3.0 is supported, with the exception of the ca
 +
 . To configure your API Proxy for this API instance you need to select *Endpoint with Proxy* in the *Managing Type* fields.
 .. Select your *Proxy Deployment Target*  from the available deployment targets displayed.
-.. Select the *Check this box if you are managing this API in Mule 4 or above* check mark.
+.. If you want to manage the API using Mule 4 or later (recommended), accept the default selection.
++
+The parameters that you must specify differ based on the Mule version you select.
```
Comment (hannanelson) [inline modules/ROOT/pages/proxy-deploy-raml-oas-proxy.adoc]:
```suggestion
The parameters that you specify differ based on the Mule version you select. 
```

File: modules/ROOT/pages/getting-started-proxy.adoc [hunk 2]
```diff
@@ -51,9 +51,9 @@ The list displays the full name of the API asset that you created in the previou
 +
 If you do not want to configure your API with proxy, select *Basic endpoint*. 
 +
-- *Mule version:* If you are managing an API for Mule 4 or later, select this checkbox. 
+- *Mule version:* If you want to manage the API using Mule 4 or later (recommended), accept the default selection.
 +
-In this example, create the API for Mule 4.
+The parameters that you must specify differ based on the Mule version you select. In this example, create the API for Mule 4.
```
Comment (hannanelson) [inline modules/ROOT/pages/getting-started-proxy.adoc]:
```suggestion
The parameters that you specify differ based on the Mule version you select. In this example, create the API for Mule 4.
```

File: modules/ROOT/pages/configure-api-task.adoc [hunk 3]
```diff
@@ -31,7 +31,9 @@ image::create-instance-endpoint-midp.png[Create and configure an API Endpoint]
 You can accept the default Path `/` for the proxy endpoint or enter another path. 
 . If you are configuring a *Basic Endpoint*, perform the following steps:
 .. In the *Application type* section, specify whether your application uses a Hybrid or CloudHub environment, or if you are running a non-Mule application managed by Anypoint Service Mesh.
-.. Select the *Mule version* box if you want to manage APIs in Mule 4 or later.
+.. If you want to manage the API using Mule 4 or later (recommended), accept the default selection.
++
+The parameters that you must specify differ based on the Mule version you select.
```
Comment (hannanelson) [inline modules/ROOT/pages/configure-api-task.adoc]:
```suggestion
The parameters that you specify differ based on the Mule version you select. 
```

File: modules/ROOT/pages/building-https-proxy.adoc [hunk 4]
```diff
@@ -33,16 +33,19 @@ If you are not familiar with the secrets manager concepts, you should also revie
 For more information, see xref:api-manager::manage-exchange-api-task.adoc[Managing an API from Exchange]. 
 +
 While configuring your API Proxy, make sure you also select the following configurations:
++
 .. Select *Endpoint with Proxy* in the *Managing Type* fields
-.. Select the *Check this box if you are managing this API in Mule 4 or above* check mark.
+.. If you want to manage the API using Mule 4 or later (recommended), accept the default selection.
++
+The parameters that you must specify differ based on the Mule version you select.
```
Comment (hannanelson) [inline modules/ROOT/pages/building-https-proxy.adoc]:
```suggestion
The parameters that you specify differ based on the Mule version you select. 
```

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/proxy-deploy-raml-oas-proxy.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-deploy-raml-oas-proxy.adoc]: Done! Thanks!

File: modules/ROOT/pages/getting-started-proxy.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/getting-started-proxy.adoc]: Done! Thanks!

File: modules/ROOT/pages/configure-api-task.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-api-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/building-https-proxy.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/pages/building-https-proxy.adoc]: Done! Thanks!

**Review by hannanelson — APPROVED** [review]

**Pull request merged**